SEED-platform/seed
# DQ button check mark sequence fix #786, alert unsaved changes #1848

#### Any background context you want to provide?
This is a joint pull request for #786 and #1848 

#### What's this PR do?
1. fixed data quality admin button check mark sequence;
2. alert users unsaved changes while switching menu taps.

#### How should this be manually tested?
For testing #786, on data quality admin page, click on each of restore, reset and save changes button.
Expect to see check mark to only appear on last clicked button to display current state.

For testing #1848, click on "create a new rule" button, or anywhere within table, without clicking "save changes", tap on any clickable taps on site.
Expect to see prompt window to alert users that setting has been modified.

#### What are the relevant tickets?
#786, #1848 

#### Screenshots (if appropriate)
<img width="1086" alt="Screen Shot 2019-12-31 at 5 33 16 PM" src="https://user-images.githubusercontent.com/49968203/71636945-6e8df380-2bf5-11ea-8f06-c946edcccd65.png">
